packages/gasket-plugin-nextjs/lib/webpack-config.js

```diff
@@ -0,0 +1,57 @@
+const isGasketCore = /@gasket[/\\](core)/;
+
+function validateNoGasketCore(ctx, callback) {
+  if (isGasketCore.test(ctx.request)) {
+    return callback(new Error('@gasket/core should not be used in browser code.'));
+  }
+  return callback();
+}
+
+/**
+ * Externalize @gasket/core in the server build.
+ * We do this to enable GASKET_ENV to be picked up and passed to the plugins.
+ * Otherwise, unique builds per environment would be required.
+ *
+ * @param ctx
+ * @param callback
+ * @returns {*}
+ */
+function externalizeGasketCore(ctx, callback) {
+  if (isGasketCore.test(ctx.request)) {
+    const externalsType = ctx.dependencyType === 'esm' ? 'module' : 'commonjs';
+    return callback(null, [externalsType, ctx.request].join(' '));
+  }
+  return callback();
+}
+
+/**
+ * Configure Next.js Webpack for @gasket/core
+ *
+ * @param {Object} gasket - The gasket API.
+ * @param {Object} webpackConfig - Initial Next.js webpack config
+ * @returns {Object} Partial webpack config with UXCore2 support.
+ * @public
+ */
+function webpackConfigHook(gasket, webpackConfig) {
+  if (webpackConfig.name === 'client') {
```

Comment:
Is `name` the best way to detect this? Shouldn't we use the `isServer` property [from the docs](https://nextjs.org/docs/app/api-reference/next-config-js/webpack)?

Reply:
I'll try it out - we want it to externalize gasket/core from edge runtime bundles also, which is another non-client named config. If `isServer` covers that then 👍